src/app/src/actions/facilities.js

```diff
@@ -58,10 +61,14 @@ export function fetchFacilities({
 
                     pushNewRoute(makeFacilityDetailLink(facilityID));
                 }
-
                 return data;
             })
-            .then(data => dispatch(completeFetchFacilities(data)))
+            .then((data) => {
+                dispatch(completeFetchFacilities(data));
+                if (setTab) {
+                    dispatch(makeSidebarFacilitiesTabActive());
+                }
```

Comment:
Moving this out of the reducer into an action makes a lot of sense. Nice update to the design 👍